docs/design/2020-06-24-placement-rules-in-sql.md

```diff
@@ -229,8 +229,8 @@ SHOW PLACEMENT;
 +----------------------------+----------------------------------------------------------------------+------------------+
 | target                     | placement                                                            | scheduling_state |
 +----------------------------+----------------------------------------------------------------------+------------------+
-| POLICY system              | PRIMARY_REGION="us-east-1" REGIONS="us-east-1,us-east-2" FOLLOWERS=4 | SCHEDULED        |
-| POLICY default             | PRIMARY_REGION="us-east-1" REGIONS="us-east-1,us-east-2"             | SCHEDULED        |
+| POLICY system              | PRIMARY_REGION="us-east-1" REGIONS="us-east-1,us-east-2" FOLLOWERS=4 |                  |
```

Comment:
@morgo What do you think about the scheduling_state for a policy. Some options: `SCHEDULED ` , `NULL` or an empty string just like this pr did, which is better? Actually we do not schedule policy in PD.

Reply:
Null is ideal (since it means not applicable)